playwright/screenshots/chrome-webstore-screenshots.spec.ts

```diff
@@ -0,0 +1,108 @@
+import { extensionViewport } from '../utils/extensionTestExtend'
+import { warnSlowApi } from '../utils/warnSlowApi'
+import { mockApiMoreData } from '../utils/mockApi'
+import { privateKey } from '../../src/utils/__fixtures__/test-inputs'
+import { test, expect, Page } from '@playwright/test'
+
+const screenshotCss = `
+  * { scrollbar-width: none; }
+  [data-testid="build-banner"] { display: none; }
+`
+
+const chromeWebStoreDimensions = {
+  width: 1200,
+  height: 800,
+}
+
+/** Scale for higher quality larger screenshots */
+const deviceScaleFactor = (chromeWebStoreDimensions.height - 20) / extensionViewport.height
+test.use({
+  deviceScaleFactor: deviceScaleFactor,
+  viewport: {
+    width: Math.round(chromeWebStoreDimensions.width / deviceScaleFactor),
+    height: Math.round(chromeWebStoreDimensions.height / deviceScaleFactor),
+  },
+})
+
+async function setup(page: Page, url: string) {
+  await page.goto('/app.webmanifest')
+  await page.setContent(`
+    <style>
+      body {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100vh;
+      }
+      iframe {
+        width: ${extensionViewport.width}px;
+        height: ${extensionViewport.height}px;
+        border: 5px solid #0500e2;
+      }
+    </style>
+    <body style="">
+      <iframe src="${url}"></iframe>
+    </body>
+  `)
+  return page.frameLocator('iframe')!
+}
+
+test.beforeEach(async ({ context }) => {
+  await warnSlowApi(context)
+  await mockApiMoreData(context)
+})
+
+test('make screenshots for Chrome Web Store', async ({ page }) => {
+  const frame = await setup(page, `/`)
+  await page.screenshot({
+    path: './screenshots/extension-store-1.png',
+    style: screenshotCss,
```

Comment:
the banner css display none is ignored in my local env :thinking: 

![Screenshot from 2024-08-27 16-15-19](https://github.com/user-attachments/assets/3980be59-4e71-4a6a-924a-b1a4cdfc7e03)

Reply:
:shrug: same on my side now. I don't know how it worked for me yesterday.

Had to update playwright and reinstall it now and it works again. Maybe it would work on CI now too